src/utils/dataset_utils.py

```diff
@@ -1056,25 +999,13 @@ def get_gold_labels_from_meta(session, candidate_class, target, split, annotator
                 label = 1
             else:
                 label = -1
-                
```

Comment:
Need to add phone bit in here (see diff)

Reply:
Done